cuda_bindings/cuda/bindings/_path_finder/README.md

```diff
@@ -0,0 +1,61 @@
+# `cuda.bindings.path_finder` Module
+
+## Public API (Work in Progress)
+
+Currently exposes two primary interfaces:
+
+```
+cuda.bindings.path_finder._SUPPORTED_LIBNAMES  # ('nvJitLink', 'nvrtc', 'nvvm')
+cuda.bindings.path_finder._load_nvidia_dynamic_library(libname: str) -> LoadedDL
```

Comment:
```suggestion
cuda.bindings._path_finder._SUPPORTED_LIBNAMES  # ('nvJitLink', 'nvrtc', 'nvvm')
cuda.bindings._path_finder._load_nvidia_dynamic_library(libname: str) -> LoadedDL
```

Reply:
I think the path is correct

Reply:
I cannot easily rename the cuda/bindings/path_finder.py → cuda/bindings/_path_finder.py because that'd clash with the name of the subdirectory.